client/src/components/DatasetInformation/DatasetInformation.vue

```diff
@@ -1,11 +1,11 @@
 <template>
     <DatasetProvider :id="hda_id" v-slot="{ item: dataset, loading }">
         <div v-if="!loading">
-            <h3>Dataset Information</h3>
+            <h3>Dataset Information (<a :href="'/datasets/edit?dataset_id=' + dataset.id" target="_top">edit</a>)</h3>
             <table id="dataset-details" class="tabletip info_data_table">
                 <tbody>
                     <tr>
-                        <td>Number</td>
+                        <td>Number in history</td>
```

Comment:
This is nice!
```suggestion
                        <td>Number in History</td>
```
Looks slightly better ?